main.js

```diff
@@ -232,7 +241,7 @@ function createImageDetails (item) {
   return imageColumn
 }
 
-function createInfoDetails (item) {
+function createInfoDetails(item) {
 
   var detailsColumn = document.createElement('div')
     detailsColumn.classList.add('col-6')
```

Comment:
let's align these lines to the correct indentation level (and also the other statement in this function):

```js
var detailsColumn = document.createElement('div')
detailsColumn.classList.add('col-6')
```

vs

```js
var detailsColumn = document.createElement('div')
  detailsColumn.classList.add('col-6')
```